python-src/resources.py

```diff
@@ -39,7 +39,8 @@ def post(self):
         game = Game()
         request_data = request.get_json()
         game.players = [ndb.Key(urlsafe=request_data['player_id'])]
-        game.cities = Util.get_cities(int(request_data.get('number_of_cities', 3)))
+        game.cities = Util.get_cities(int(request_data.get('number_of_cities', 3)),
```

Comment:
when you create a game, you can send a new parameter 'difficulty' which can be either 0 for easy or 1 for hard. this has to be implemented in the game service, that when you create a game, not only player ID but also the difficulty is passed as POST data

there is an example on how the post request has to look like in postman